src/App.js

```diff
@@ -5,18 +5,22 @@ import Layout from './components/common/Layout';
 import CssBaseline from '@mui/material/CssBaseline';
 import { ThemeProvider } from '@mui/material/styles';
 import store from './redux/store';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
 
 // import Loader from './components/common/Loader';
 
 function App() {
   return (
-    <ThemeProvider theme={theme}>
-      <CssBaseline />
-      <Provider store={store}>
-        <Layout />
-      </Provider>
-      {/* <Loader /> */}
-    </ThemeProvider>
+    <DndProvider backend={HTML5Backend}>
```

Comment:
pretty sure we don't need this here but we can move it to wrap only the visualise layout.